content/02.delete-me.md

```diff
@@ -1,15 +1,146 @@
-Content is written in [Pandoc Markdown](http://pandoc.org/MANUAL.html#pandocs-markdown) source files.
-See [`USAGE.md`](https://github.com/greenelab/manubot-rootstock/blob/master/USAGE.md) for more information on how to use the Manubot.
 
-The Manubot project began with the [Deep Review](https://github.com/greenelab/deep-review), where it was used to compose a highly-collaborative review article [@doi:10.1098/rsif.2017.0387].
-Another example manuscript that was created with Manubot is:
 
-+ The Sci-Hub Coverage Study ([GitHub](https://github.com/greenelab/scihub-manuscript), [HTML manuscript](https://greenelab.github.io/scihub-manuscript/)) [@doi:10.7554/eLife.32822]
+## Examples
 
-If you notice a problem with Manubot, it's best to submit an upstream fix to the appropriate repository:
-[`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock) for the git repository stub or [`greenelab/manubot`](https://github.com/greenelab/manubot) for the Python package.
+Below is a full list of formatting elements for `.html` and `.pdf` exports that are supported by the Manubot-provided themes. 
+Compare the input Markdown (`.md` files in the `/content` directory) to the output you see below.
+
+# Heading 1
+
+## Heading 2
+
+### Heading 3
+
+#### Heading 4
+
+##### Heading 5
+
+###### Heading 6
+
+<!-- an arbitrary comment. not visible in output. -->
+
+Horizontal rule:
+
+---
+
+A long paragraph of text.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+Another long paragraph.
+Donec massa sapien faucibus et molestie ac feugiat sed lectus.
+Est sit amet facilisis magna etiam.
+Ornare arcu dui vivamus arcu felis bibendum.
+Magna fermentum iaculis eu non diam.
+Vehicula ipsum a arcu cursus vitae.
+Ante metus dictum at tempor commodo ullamcorper a.
+Vitae tortor condimentum lacinia quis vel eros.
+Ut venenatis tellus in metus vulputate eu scelerisque felis imperdiet.
+Facilisis sed odio morbi quis.
+Enim ut sem viverra aliquet eget sit amet tellus.
+
+*Italic* _text_
+
+**Bold** __text__
+
+Combined *italics and __bold__*
+
+~~Strikethrough~~
+
+Line break without starting new paragraph by putting  
+two spaces at end of line
+
+1. Ordered list item
+2. Ordered list item
+    a. Sub-item
+    b. Sub-item
+        i. Sub-sub-item
+3. Ordered list item
+    a. Sub-item
+
+* List item
+- List item
++ List item
+
+superscript<sup>2</sup>
+
+subscript<sub>2n+1</sub>
+
+[Link](https://www.google.com)
+
+[Link with title (hover) text](https://www.google.com "Google")
+
+[Link by reference][google homepage]
```

Comment:
Capitalization difference here and two lines below. I guess this is okay? but perhaps not preferred.

Would be great to change link from Google to something more project related like manubot.org.

Also should do a bare URL like `<https://manubot.org>`: without <> the URL is not hyperlinked to itself.

Reply:
Will definitely change to manubot.org or similar and put a bare URL.

Regarding the capitalization difference, I think I did that just to show that it wasn't case sensitive. It's up to you. We don't necessarily have to show everything that's possible in markdown (like the lists being able to be `-` `*` or `+`), we can choose our preferred thing and just show that. It is just a template after all; doesn't have to be a complete markdown tutorial (though I do think giving a quick intro is good for people who've never seen it before so they get the general gist and can look up more about it if they want to).